pytorch_lightning/trainer/connectors/checkpoint_connector.py

```diff
@@ -299,39 +299,11 @@ def restore_lr_schedulers(self) -> None:
     # PRIVATE OPS
     # ----------------------------------
 
-    def hpc_save(self, folderpath: str, logger: Optional[LightningLoggerBase]) -> str:
-        # make sure the checkpoint folder exists
+    def hpc_save_path(self, folderpath: str) -> str:
```

Comment:
super nit: you could make this, `max_ckpt_version_in_folder`, and `get_max_ckpt_path_from_folder` staticmethods. that's usually a sign that this class can be further streamlined by moving those out to utility functions. that could be for a separate issue/PR